slack_bot.py

```diff
@@ -37,14 +29,113 @@ def post_messages_to_slack(self, messages: List[str]) -> SlackResponse:
         )
 
     # For the filtered data, generate messages and send them to slack
-    def execute_for_codes(self, covid_data: Dict) -> SlackResponse:
+    def execute_for_covid_data(self, covid_data: Dict) -> SlackResponse:
         # Generate the messages for each state/country code
         messages: List[str] = []
         for code_data in covid_data:
             messages.append(self.generate_message_for_code(covid_data[code_data]))
 
+        blocks = [
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": message
+                }
+            } for message in messages
+        ]
+
         # Post these messages to slack
-        return self.post_messages_to_slack(messages)
+        return self.post_messages_to_slack(blocks)
+
+    def execute_for_vax_stats(self, vax_data: Dict) -> SlackResponse:
+        # Generate the messages for each state/country code
+        blocks = [
+            {
+                "type": "header",
+                "text": {
+                    "type": "plain_text",
+                    "text": ":dart: Vax Stats"
+                }
+            },
+            {
+                "type": "divider"
+            }
+        ]
+        
+        for code in vax_data:
+            blocks.extend(self.generate_vax_stats_for_code(vax_data[code]))
```

Comment:
```suggestion
        ].extend( [self.generate_vax_stats_for_code(vax_data[code]) for code in vax_data] )
```
Nitpicky, but this is a little more pythonic :)